tensorboard/uploader/exporter.py

```diff
@@ -180,19 +227,120 @@ def _request_scalar_data(self, experiment_id, read_time):
             metadata = base64.b64encode(
                 response.tag_metadata.SerializeToString()
             ).decode("ascii")
-            wall_times = [
-                t.ToNanoseconds() / 1e9 for t in response.points.wall_times
-            ]
-            yield {
+            json_data = {
                 u"run": response.run_name,
                 u"tag": response.tag_name,
                 u"summary_metadata": metadata,
-                u"points": {
-                    u"steps": list(response.points.steps),
-                    u"wall_times": wall_times,
-                    u"values": list(response.points.values),
-                },
             }
+            filename = None
```

Comment:
Does this cause a `KeyError` in the caller if the server returns a
response with neither `points` nor `blob_sequences` set—say, because
we’ve added tensor storage and it returns `tensors` instead?

Should we perhaps instead warn in that case and not yield, since the
`json_data` is unfinished and there’s no filename to write, anyway?